modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCheckout.testcase

```diff
@@ -0,0 +1,241 @@
+@component-name = "portal-commerce"
+definition {
+
+	property custom.properties = "jsonws.web.service.paths.excludes=";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
```

Comment:
This property can be removed as I'm not seeing any usage of the private pages on the tests